velox/exec/Task.cpp

```diff
@@ -1817,7 +1813,7 @@ StopReason Task::leaveSuspended(ThreadState& state) {
       }
       if (terminateRequested_) {
         state.isTerminated = true;
-        return StopReason::kTerminate;
+        return StopReason::kAlreadyTerminated;
```

Comment:
Why is this change?

Reply:
I reverted. But it is a little bit tricky to tell the difference between the above two cases handlings.